apidef/api_definitions.go

```diff
@@ -351,6 +351,7 @@ type APIDefinition struct {
 	JWTDisableExpiresAtValidation bool                 `bson:"jwt_disable_expires_at_validation" json:"jwt_disable_expires_at_validation"`
 	JWTDisableNotBeforeValidation bool                 `bson:"jwt_disable_not_before_validation" json:"jwt_disable_not_before_validation"`
 	JWTSkipKid                    bool                 `bson:"jwt_skip_kid" json:"jwt_skip_kid"`
+	JWTExpiryOverridesPolicy      bool                 `bson:"jwt_expiry_overrides_policy" json:"jwt_expiry_overrides_policy"`
```

Comment:
@buger @josephearl 

Should this be configurable? If JWT contains an expiry - surely we should just use the JWT's expiry without looking at the Policy expiry?

Reply:
The thinking with configurability is that it wont disrupt existing flows for users with JWT.

Reply:
Agree with that - but the logic is currently bugged IMO - this is supposed to resolve the issue, not give user option to resolve the issue.

Reply:
That's why I tagged @buger for his opinion

Reply:
Probably agree with @asoorm. In this case we solving a bug.